robot/Cumulus/tests/browser/crlps/last_soft_credit_opportunity.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/CustomizableRollupsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/ObjectMangerPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Delete Records and Close Browser
+
+***Keywords***
+# creates test data a contact and an opportunity for the contact
+Setup Test Data
+    Setupdata   contact   ${contact1_fields}     ${opportunity_fields}
+    ${date} =   Get Current Date       result_format=%-m/%-d/%Y
```

Comment:
I am not sure if you have any windows users the format with a prefix `-` will not work on windows. If you are interested we can discuss on how to handle these date formatting.

Reply:
we dont have any windows users so I guess I can leave it as it is for now due to time constraint but would love to sync up on how to handle for future changes

Reply:
Yes, you do not have to take care of this in this PR. Sorry, I should have mentioned that.